vnext/docs/api/react-native-windows.checkbox.md

```diff
@@ -0,0 +1,18 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [react-native-windows](./react-native-windows.md) &gt; [CheckBox](./react-native-windows.checkbox.md)
+
+## CheckBox class
+
+<b>Signature:</b>
+
+```typescript
+export declare class CheckBox extends React.Component<ICheckBoxProps> 
+```
+
+## Methods
+
+|  Method | Modifiers | Description |
+|  --- | --- | --- |
+|  [render()](./react-native-windows.checkbox.render.md) |  |  |
```

Comment:
A fair number of render() methods are being generated.  This doesn't seem useful... Is there a way to suppress these?